app/services/idv/steps/ssn_step.rb

```diff
@@ -29,6 +30,12 @@ def invalid_state_response
         mark_step_incomplete(:document_capture)
         FormResponse.new(success: false)
       end
+
+      private
+
+      def threatmetrix_session_id
+        SecureRandom.uuid
+      end
```

Comment:
Can we guard it with a feature flag here as well so that we don't put a session ID on the page if the flag is off?